packages/buidler-core/src/internal/buidler-evm/provider/modules/evm.ts

```diff
@@ -32,6 +41,29 @@ export class EvmModule {
     throw new MethodNotFoundError(`Method ${method} not found`);
   }
 
+  // evm_setNextBlockTimestamp
+
+  private _setNextBlockTimestampParams(params: any[]): [number] {
+    return validateParams(params, t.number);
```

Comment:
Oh, also here. I'd use a quantity value.

Reply:
This is to keep the consistency with `_increaseTimeParams` API. It would be best if we can support both hex and base 10 params here but I think `number` would be ok for now.

Reply:
It checks to ensure `nextBlockTimestamp` is set. If it is not (value is 0), it means the next block time Is set and it set the next block timestamp to `nextBlockTimestamp`.